docs/plots/pylab_backend_area.rst

```diff
@@ -0,0 +1,20 @@
+*****************************************
```

Comment:
This file should be renamed to `pylab_area_backend` (e.g see line 333 of `docs/plots/index.rst` and line 2 of this file)